src/frontend/src/optimizer/plan_node/stream_expand.rs

```diff
@@ -38,16 +38,18 @@ impl StreamExpand {
             | Distribution::UpstreamHashShard(_, _) => Distribution::SomeShard,
             Distribution::Broadcast => unreachable!(),
         };
+        let schema = logical.schema().clone();
+        let mut watermark_columns = FixedBitSet::with_capacity(schema.len());
+        watermark_columns &= logical.input().watermark_columns();
```

Comment:
the schema of `Expand` is that 
`| expanded columns(i.e. some columns are set to null) | original columns of input | flag |`
So the watermark_columns should be right shifted with `logical.input().schema().len` (but I do not find the right shift method of Fixedbitset)